modules/local/atacreads/macs2.nf

```diff
@@ -0,0 +1,50 @@
+// Import generic module functions
```

Comment:
Continuing discussion https://github.com/nf-core/hicar/pull/1#discussion_r741521686

> Because the central one does not support BED format. This can be changed once the central one have BED format support.

Can we update the version in nf-core/modules and use that here instead of having a local module? Maybe there is already something in progress for this, but I couldn't find it when I just did a quick search..

Reply:
That will be great if we have that one. Once it is possible, I move to nf-core/modules.

Reply:
@jianhong - are you able to make a pull request to nf-core/modules with this local module then, so that the central copy is updated with your additions?

Reply:
I created a pr just now.

Reply:
x-ref https://github.com/nf-core/modules/pull/1038